pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Pull Requests
+
+Please be sure you have read our documentation regarding contributing
+here: https://docs.pulpproject.org/index.html
+
+There you can read a Contribution Checklist and learn more about
+appropriate branches and rebasing among other things.
+
+Once you are ready to create a pull request:
+
+- [ ] If there is a bug for this issue, please title the pull request
+"<bug_id> - Short Message"
+- [ ] In the commit message or comment section for your pull request,
+please include a link to the issue.
```

Comment:
For commit message, you might want to link to our guide on how to format commit messages:

https://docs.pulpproject.org/en/3.0/nightly/contributing/git.html#commit-message

Reply:
Good idea.